docs/information/docker.md

```diff
@@ -30,12 +36,44 @@ docker run \
 * `--device=/dev/ttyACM0`: Location of adapter (e.g. CC2531)
 * `-v /run/udev:/run/udev:ro --privileged=true`: is optional, only required for autodetecting the port
 * Optional: in case your MQTT broker is running on `localhost` and is not within the same Docker network as the Zigbee2MQTT container also add `--network host \`.
+* Note: Running in "--privileged" mode is a security risk.
+
+## Running as non-root
+
+1. Identify your device:
+```
+$ ls -l /dev/serial/by-id
+```
+
+2. Identify the group that has access to the device (in Ubuntu, e.g. it might be assigned to "dialout"):
+```
+$ ls -l /dev/tty*
+```
+
+3. Check the user&group id you want to execute the docker image with:
+```
+$ id
+```
+
+4. Start the docker container (note: interface, user&group ID must match the outputs above, priveleged mode is not required):
```

Comment:
It's not necessary to mention things that are not required, I suggest to remove `priveleged mode is not required`